modules/representation_management/lib/representation_management/v0/pdf_constructor/form_2122a.rb

```diff
@@ -159,7 +159,7 @@ def representative_identification(data)
             "#{PAGE1_KEY}.Last_Name[0]": data.representative.last_name,
             # Representative Type
             "#{PAGE1_KEY}.RadioButtonList[0]": representative_type_checkbox(
-              data.representative.individual_type.to_s.upcase
+              data.representative_individual_type.to_s.upcase
```

Comment:
Might need to update the mapping to account for two different claims agent string options.
- AccreditedIndividual uses `claims_agent`
- Veteran::Service::Representative uses `claim_agents`

Reply:
It looks like I have to convert both `claims_agent` and `claim_agents` to just `agent` to get the checkbox filled on the PDF.  This was a great catch!

Reply:
Yep, confirmed.  `representative_individual_type` needs to return `agent` for the PDF checkbox to be filled.